src/base/net/smtp.cpp

```diff
@@ -148,8 +148,7 @@ void Smtp::sendMail(const QString &from, const QString &to, const QString &subje
     // Encode the body in base64
     QString crlfBody = body;
     const QByteArray b = crlfBody.replace(u"\n"_s, u"\r\n"_s).toUtf8().toBase64();
-    const int ct = b.length();
-    for (int i = 0; i < ct; i += 78)
+    for (int i = 0; i < b.length(); i += 78)
```

Comment:
Why did you change it? Previously, you insisted that repeated getter calls be replaced by assigning a value to a constant and using it.
Now you're making the opposite change yourself...

Reply:
>Now you're making the opposite change yourself...

please, you sound a bit dramatic...

>you insisted that repeated getter calls be replaced by assigning a value to a constant and using it.

I don't recall we have such rule: "invariant in for loop conditionals have to be a separate variable". Also I don't recall you making such rule. Are you initiating now?

Anyway, I moved it around.

Reply:
> I don't recall we have such rule: "boundary index in loop conditional need to be a separate variable". Also I don't recall you making such rule. Are you initiating now?

I'm not. I am against the rules governing "not so important" aspects. If there is no strict justification for requiring something, we should not do it.